examples/example-workload/httpproxy/README.md

```diff
@@ -0,0 +1,5 @@
+# HTTPProxy
+
+This directory contains example code for run contour with HTTPProxy.
+
+Most subdirectories contain a (complete) set of Kubernetes YAML that can be applied to a cluster.
```

Comment:
I think we can drop this sentence, it seems more relevant to subdirs that have YAML for _installing_ Contour.

Reply:
done